python-package/lightgbm/basic.py

```diff
@@ -1100,7 +1100,7 @@ def predict(
 
         Parameters
         ----------
-        data : str, pathlib.Path, numpy array, pandas DataFrame, pyarrow Table or scipy.sparse
+        data : str, pathlib.Path, numpy array, pandas DataFrame, scipy.sparse or pyarrow Table
```

Comment:
I am fine with either ordering, but just curious... why did you switch the order of these? For consistency with the type hint?

Reply:
> For consistency with the type hint?

Yep. I can revert this reordering change to simplify the diff.

Reply:
No need to! I'm ok with this change, just wanted to understand so I can match that pattern in my own PRs in the future.